components/raftstore/src/store/worker/region.rs

```diff
@@ -427,8 +414,12 @@ where
             coprocessor_host: self.coprocessor_host.clone(),
         };
         s.apply(options)?;
-        self.coprocessor_host
-            .post_apply_snapshot(&region, peer_id, &snap_key, Some(&s));
+        if let Err(e) =
+            self.coprocessor_host
+                .post_apply_snapshot(&region, peer_id, &snap_key, Some(&s))
+        {
+            return Err(box_err!("post apply snapshot error {:?}", e));
```

Comment:
Returning error can make the applying snapshot in undetermined state. I'm afraid it can be misused.

Reply:
we log an error here instead